editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/queryResults/QueryResults.tsx

```diff
@@ -243,7 +243,7 @@ export default function QueryResults() {
 					onSync={handleSync}
 					importedItemsCount={importedItems.length}
 					unImportedItemsCount={
-						(data?.total ?? 0) -
+						(data?.items.length ?? 0) -
```

Comment:
This still seems buggy in its own way to me.. `data.items.length` can't exceed `DEFAULT_ITEMS_PER_PAGE`. Each query is limited to that number (13 right now), so the max of that length equals that number. We don't add it up, but reset it each time new items are loaded right now.  
Adding it up with each lazy load doesn't solve this either. Would only tell you the whole story once you've scrolled to EOS.

Now, we definitely want the `data.total` to be the basis of the `Import All` count. From there I can imagine a lazy and a probably accurate way:
1. Lazy but easy  
 Just leave it at `data.total` and add a tooltip saying "Imports up to {the indicated number} items but will skip already imported ones".
2. Tracker-id based calculation
 To get the count of already imported items, don't check their ids, but their **tracker's id**. You have that from the get-go on the Import modal, no need to have loaded all items. So if you also add that id as metadata to the cards, you just subtract the number of already imported items in that tracker.

Reply:
tracker-id based calculation is a very clever and neat solution thank you